GabrielaRamirez/EjerciciosHackerrank/src/challenge/Main.java

```diff
@@ -0,0 +1,20 @@
+package challenge;
+
+public class Main {
+
+    public static void main(String[] args){
+        Icecream icecream = new Icecream();
+        System.out.println(icecream.BRAND);
+        icecream.printFlavour("coconut");
+        icecream.topping("chocolate");
+
+
+        Nieve nieve = new Nieve();
```

Comment:
nieve??? paleta??  Java-nou-hablo-espaniol

Only coding in english is a good practice not only in Java

Reply:
XD ok, i'll change